tests/utils/mod.rs

```diff
@@ -96,7 +96,11 @@ async fn create_test_table(large_utf8: bool, dict_encoded: bool) -> Result<Sessi
         Arc::new(Schema::new(vec![
             Field::new("json_data", DataType::Utf8, false),
             Field::new("str_key1", DataType::Utf8, false),
-            Field::new("str_key2", DataType::Utf8, false),
+            Field::new(
```

Comment:
With this change to the test suite and not the corresponding changes to `common.rs`, the issue in #53 reproduced as part of the suite.